variants/deploy_with_capistrano/template.rb

```diff
@@ -134,24 +134,22 @@
 
 gsub_file!("config/deploy.rb", old_generated_cap_config_snippet, new_ackama_cap_config_snippet)
 
-insert_into_file! "Capfile", after: /install_plugin Capistrano::SCM::Git/ do
-  <<~EO_RUBY
-    # Include tasks from other gems included in your Gemfile
-    #
-    # For documentation on these, see for example:
-    #
-    #   https://github.com/capistrano/rbenv
-    #   https://github.com/capistrano/bundler
-    #   https://github.com/capistrano/rails
-    #   https://github.com/sheharyarn/capistrano-rake
-    #
-    require "capistrano/rbenv"
-    require "capistrano/bundler"
-    require "capistrano/rails/assets"
-    require "capistrano/rails/migrations"
-    require "capistrano/rake"
-  EO_RUBY
-end
+gsub_file!("Capfile", '# require "capistrano/rbenv"', 'require "capistrano/rbenv"')
+gsub_file!("Capfile", '# require "capistrano/bundler"', 'require "capistrano/bundler"')
+gsub_file!("Capfile", '# require "capistrano/rails/assets"', 'require "capistrano/rails/assets"')
+gsub_file!("Capfile", '# require "capistrano/rails/migrations"', 'require "capistrano/rails/migrations"')
+
+insert_into_file!(
+  "Capfile",
+  "    #   https://github.com/sheharyarn/capistrano-rake",
+  after: "#   https://github.com/capistrano/passenger"
+)
+
+insert_into_file!(
+  "Capfile",
+  'require "capistrano/rake"',
```

Comment:
This will need removing once https://github.com/ackama/rails-template/pull/619 is merged, since it will continue to insert into the file.